src/applications/claims-status/actions/index.jsx

```diff
@@ -171,7 +172,7 @@ export function pollClaimsStatus({
       }
 
       setTimeout(
-        pollClaimsStatus,
+        pollRequest,
         pollingInterval,
         { onError, onSuccess, pollingInterval, request, shouldFail, shouldSucceed }
```

Comment:
Does `target` not need to be passed to the delayed call to `pollRequest`?

Reply:
whoops